scripts/core/EXT_Exp_CounterBasedEvents.rst

```diff
@@ -0,0 +1,49 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+.. _ZE_experimental_event_counter_based:
+
+=========================
+ Counter-Based Events Extension
+=========================
+
+API
+----
+ 
+* Enumerations
+
+    * ${x}_event_counter_based_exp_flags_t
+    * ${x}_event_counter_based_exp_version_t
+
+* Structures
+
+    * ${x}_event_counter_based_exp_desc_t
+
+Counter-Based Events
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, events in Level Zero contains one of two states: signaled or not signaled.
+Signaling of an event, as well as resetting its state, can be done from either host or
+device.
+
+Another way of tracking the state of an event is by using a counter, which is incremented
+every time a task has completed in the device. Using a counter-based event may be
+provide Level Zero driver implementations with the opportunity for both
```

Comment:
statement reformatting: " may be provide" to "may provide"

Reply:
thanks, changed.